000-index.md

```diff
@@ -2,3 +2,4 @@
 
 * [Public C++ Streams](001-public-stream-base.md)
 * [ES6 Module Interoperability](002-es6-modules.md)
+* [stream.Readable#readInto()](003-readable-read-into.md)
```

Comment:
Could you remove this line and rename `003-*` to `XXX-*`? Whoever merges the PR changes those, since another might land before this and all. :)